en/docs/license.md

```diff
@@ -0,0 +1,6 @@
+# Ballerina VS Code extesnion license 
+
+WSO2 grants you a worldwide, non-exclusive right to copy and use the Ballerina Visual Studio Code extenstion in 
```

Comment:
Typo `extenstion`

Reply:
Fixed in https://github.com/wso2/docs-ballerina-visual-studio-code-plugin/pull/21/files.